distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -375,16 +374,22 @@ public void beforeTest() throws Exception {
               featureName
                   + "=\"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin\"");
 
-      List<String> servicePolicies = new ArrayList<>();
+      List<String> servicePolicies = new ArrayList<>(SERVICES_TO_FILTER);
       servicePolicies.addAll(SERVICES_TO_FILTER);
       servicePolicies.replaceAll(
           serviceName ->
               serviceName
                   + "=\"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin\"");
 
-      properties.put("featurePolicies", featurePolicies.stream().toArray(String[]::new));
-      properties.put("servicePolicies", servicePolicies.stream().toArray(String[]::new));
-      config.update(properties);
+      String adminConfigPolicyPid = "org.codice.ddf.admin.config.policy.AdminConfigPolicy";
```

Comment:
✏️ Can we move this into a method that configures this service like you did for the MetacardValidityFilterPlugin ?

Reply:
I thought about it but it was only being used in one place. I suppose that's the same story for Authz though. I'll move it up there, we may want to break them up to be more specific one day: `ConfigureValidationTestCommons`, `ConfigureSecurityTestCommons`, etc.